### DISCUSSION TITLE
Fail fast while fetching GoogleAds reports

### DISCUSSION THREAD

**Discussion by yershovkiril** [google/ads-api-report-fetcher#16] [Ideas]
Currently AdsReportFetcher is handling any GoogleAdsException while loading data by just printing error to the log, so we can't be really confident that data is fully loaded. 

Please consider introducing some additional flag, so that I can enforce failing of the loader in case of issues with loading.

https://github.com/google/ads-api-report-fetcher/blob/9bbb2fb1e4b3aa2268cc3e9c43478c80eb6f985a/py/gaarf/query_executor.py#L86

**Comment by AndreyMarkinPPC** [comment]
Fixed  - https://github.com/google/ads-api-report-fetcher/blob/main/py/gaarf/query_executor.py#L95
